src/LogBackend.cxx

```diff
@@ -62,28 +71,56 @@ SetLogThreshold(LogLevel _threshold) noexcept
 	log_threshold = _threshold;
 }
 
+template <typename DurationType>
+inline constexpr const char *date_format_of = "";
+
+template<>
+inline constexpr const char *date_format_of<std::chrono::minutes> = "{:%b %d %H:%M} : ";
+template<>
+inline constexpr const char *date_format_of<std::chrono::seconds> = "{:%b %d %H:%M:%S} : ";
+template<>
+inline constexpr const char *date_format_of<std::chrono::milliseconds> = "{:%b %d %H:%M:%S} : ";
```

Comment:
Wow, that must be the most complicated way to do this. What's the reason for making this so complicated.